applications/automatic-post-editing.md

```diff
@@ -3,5 +3,53 @@ nav_order: 5
 parent: Application areas
 title: Automatic post-editing
 description: A method to correct errors in the machine translation output
-layout: coming_soon
 ---
+
+**Automatic post-editing** (APE) is the task of correcting automatically the output of a machine translation system.
+
+
+### Training
+Training automatic post-editing models requires access to datasets containing triplets with
+- a source text
+- the machine-translated version of this source text
+- the post-edited version of the machine-translated text
+After being trained on these triplets, automatic post-editing models can automatically detect errors and implement corrections.
+
+
+### Evolution
+The evolution of automatic post-editing models is similar to that of machine translation. The first systems were based on rules, then statistical methods were employed before moving to neural networks.
+The field gained considerable popularity after the introduction of an APE shared task at [WMT](/more/associations/wmt.md). The shared task provided triplets for training, which facilitated research.
+
+
+### Challenges
+Given that machine translation and automatic post-editing models use similar technology, the margin for improvement is often tight, which raises two main challenges.
+Namely, automatic post-editing models 
+- Must detect the right errors.
+- Must avoid overcorrections (unnecessary changes).
+
+
+### Data
+Data scarcity is another challenge for the field of automatic post-editing, since finding triplets can be a tedious task. A good source of data is the APE shared task at WMT. It includes several domains and language pairs. Since 2019, the machine translation outputs provided in the datasets are based on neural machine translation only, so it is a more accurate representation of current technology.
+
+
+Due to data availability issues, it is common to build synthetic datasets to train automatic post-editing models. Synthetic data can be created for this task by 
```

Comment:
I'd collapse this whole section.  It's also a bit subjective, e.g. "common" is probably more true in research than in industry.

Something like:

> Synthetic data can be created from parallel data.  For example, [eSCAPE (Synthetic Corpus for Automatic Post-Editing)](https://aclanthology.org/L18-1004.pdf) creates synthetic data by inserting a machine translation from a random system.